go-packages/meep-data-model/api/swagger.yaml

```diff
@@ -20,7 +20,28 @@ definitions:
       cellId:
         type: string
         description: The E-UTRAN Cell Identity as defined in ETSI TS 136 413 including the ID of the eNB serving the cell
-    description: Cellular POA configuration information
+    description: '**DEPRECATED** As of release 1.6.0, renamed to poa4GConfig'
```

Comment:
Actually the comment came that we should have gone with those 4G/5G naming before and that we were fixing an issue from the past. Regarding CELLULAR only, it could have bring confusion whether it is 4G or another technology.